### ISSUE TITLE
[Request]: "Fill" Function for Coloring Tool

### ISSUE THREAD

**Issue by Culain** [Turing-Complete-Game/Suggestions-and-Issues#521]
### Request?

Hi,

can we add a Fill-Tool to color all wires that are connected to the selected node or wire (or component)?

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by MegaIng** [comment]
This exists via `CTRL+E/R/F`, check the options menu

**Comment by Culain** [comment]
yes i was looking for exactly these functions. I guess in the control settings they are hidden too well :S
so maybe add an icon to the toolbox on the top left of the editor?

**Comment by GufNZ** [comment]
Re these functions - it took me a while to realise they only affect what you are pointing at - this could be made clearer in docs somewhere...
